examples/all-clusters-app/esp32/main/DeviceCallbacks.cpp

```diff
@@ -94,10 +130,7 @@ void DeviceCallbacks::OnInternetConnectivityChange(const ChipDeviceEvent * event
         ESP_LOGI(TAG, "Server ready at: %s:%d", event->InternetConnectivityChange.address, CHIP_PORT);
         wifiLED.Set(true);
 
-        if (chip::Mdns::ServiceAdvertiser::Instance().Start(&DeviceLayer::InetLayer, chip::Mdns::kMdnsPort) != CHIP_NO_ERROR)
-        {
-            ESP_LOGE(TAG, "Failed to start mDNS advertisement");
-        }
+        StartMdnsAdvertise(true);
```

Comment:
please add a `/* isIpv4 */` after true, since just a parameter of true is not obvious in meaning.